packages/fx-core/src/core/middleware/localSettingsLoader.ts

```diff
@@ -53,7 +55,12 @@ export const LocalSettingsLoaderMW: Middleware = async (
     if (exists) {
       ctx.localSettings = await localSettingsProvider.loadV2(ctx.contextV2?.cryptoProvider);
     } else {
-      ctx.localSettings = localSettingsProvider.initV2(hasFrontend, hasBackend, hasBot);
+      ctx.localSettings = localSettingsProvider.initV2(
+        hasFrontend,
+        hasBackend,
+        hasBot,
+        hasSimpleAuth
```

Comment:
"hasSimpleAuth" should be the fifth parameter I think.

Reply:
Fixed. Thanks!